roles/ovirt-host-deploy-firewalld/vars/firewalld_rules_common.yml

```diff
@@ -0,0 +1,9 @@
+host_deploy_firewalld_rules:
+  ## Service definition:
+  - service: rpc-bind
+  - service: snmp
+  - service: cockpit
+
+  ## Port definition:
+  # ovirt-imageio-daemon
+  - port: 54322/tcp
```

Comment:
Ahh, sorry, my mistake I've looked at systemd service file instead of firewalld configuration
Nir, is it possible to add firewalld service configuration into ovirt-imageio-daemon for oVirt 4.2?
@nirs